docs/CHANGELOG.asciidoc

```diff
@@ -38,6 +39,67 @@ This section summarizes the changes in each release.
 * <<release-notes-7.0.0-alpha1>>
 
 --
+[[release-notes-7.8.1]]
+== {kib} 7.8.1
+
+See also <<breaking-changes-7.8,breaking changes in 7.8>>.
+
+[float]
+[[enhancement-7.8.1]]
+=== Enhancements
+Monitoring::
+* Addresses a few longstanding issues with APM Stack Monitoring {pull}69052[#69052]
+SIEM::
+* Eliminates Superfluous Untitled Timelines {pull}65783[#65783]
+
+[float]
+[[bug-v7.8.1]]
+=== Bug fixes
+Alerting::
+* Hides ServiceNow and Jira from alert actions {pull}69303[#69303]
```

Comment:
I would remove this one - not sure there is much point advertising it in release notes...